pkgs/by-name/ol/ollama/package.nix

```diff
@@ -126,6 +126,7 @@ let
     # until these llama-cpp binaries can have their runpath patched
     "--suffix LD_LIBRARY_PATH : '${addDriverRunpath.driverLink}/lib'"
   ] ++ lib.optionals enableRocm [
+    "--suffix LD_LIBRARY_PATH : '${rocmPath}'"
```

Comment:
I think this used to be `makeLibraryPath [ rocmPath ]`

Reply:
Thank you! Fix is at #324206.